docs/gitbook/usage/metrics.md

```diff
@@ -477,3 +477,40 @@ spec:
     secretRef:
       name: graphite-basic-auth
 ```
+
+## Google CLoud Monitoring (Stackdriver)
+
+Create a secret that contains your project-id and your [service account json](https://cloud.google.com/docs/authentication/production#create_service_account)
+(The json isn't needed if you are running on GKE).
```

Comment:
This is not accurate, please link to the GKE docs for Workload Identity and explain how to bind it to Flagger service account. See here an example: https://fluxcd.io/docs/guides/image-update/#gcp-container-registry

Reply:
Even without Workload Identity enabled on GKE, the default access scope gives access to the Cloud logging and monitoring API. So it still works